src/SourceBridgeClient.ts

```diff
@@ -84,15 +105,20 @@ export class SourceBridgeClient {
   }
 
   private parseEnvelope(data: unknown): Partial<ResponseEnvelope> | null {
+    if (!data || typeof data !== 'string') {
+      return null
+    }
+    if (data[0] !== '{') {
```

Comment:
Is this necessary - shouldnt it be caught in the JSON parse all the same?

Reply:
Not strictly necessary, no.

Reply:
Removed.